client/src/components/TagField.js

```diff
@@ -11,38 +11,60 @@ import url from 'url';
 import debounce from 'debounce-promise';
 import PropTypes from 'prop-types';
 
-class TagField extends Component {
-  constructor(props) {
-    super(props);
+const TagField = ({
+  lazyLoad = false,
+  options,
+  creatable = false,
+  multi = false,
+  disabled = false,
+  labelKey = 'Title',
+  valueKey = 'Value',
+  SelectComponent = Select,
+  AsyncCreatableSelectComponent = AsyncCreatableSelect,
+  AsyncSelectComponent = AsyncSelect,
+  CreatableSelectComponent = CreatableSelect,
+  value: initialValue,
+  onChange,
+  optionUrl,
+  onBlur,
+  ...passThroughAttributesFromProps
```

Comment:
The old `passThroughAttributes` included `onChange`, `optionUrl`, `onBlur`, and `value` which have been explicitly pulled out above.